<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Faultdialog</class>
 <widget class="QDialog" name="Faultdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>12</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>12</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1 - 绝缘故障</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 - 输出连接器过温故障</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 - 连接器故障</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4 - 电池组温度过高</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5 - 高压继电器故障</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6 - 检测点2电压检测故障</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7 - 电流过大</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>17</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>枪号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>解除故障</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>10</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>发生故障</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>431</width>
     <height>451</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>0</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string>等级</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>枪号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>故障码</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>说明</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
